registration/conf/registration.yaml

```diff
@@ -277,7 +277,7 @@ Resources:
   RegistrationLaunchConfig:
     Type: AWS::AutoScaling::LaunchConfiguration
     Properties:
-      AssociatePublicIpAddress: True
+      AssociatePublicIpAddress: false
```

Comment:
Why is `True` titlecase and `false` lowercase?

Reply:
The reason I made it lower case is that throughout all our cloudformations for boolean we are using lowercase `true` & `false` so to make it consistent I made this lowercase as well

Also, in AWS cloud formation examples I found, for boolean lowercase `true` & `false` is used. Although, both lowercase and uppercase are accepted when you update the stack, I think lowercase is the most common